yarn.lock

```diff
@@ -6086,6 +6086,11 @@ node-fetch@2.6.0, node-fetch@^2.2.0:
   resolved "https://registry.yarnpkg.com/node-fetch/-/node-fetch-2.6.0.tgz#e633456386d4aa55863f676a7ab0daa8fdecb0fd"
   integrity sha512-8dG4H5ujfvFiqDmVu9fQ5bOHUC15JMjMY/Zumv26oOvvVJjM67KF8koCWIabKQ1GJIa9r2mMZscBq/TbdOcmNA==
 
+node-fetch@^2.6.1:
```

Comment:
Unfortunately upgrading dependencies is not so straight forward in our setup. You need to:
1. call  `yarn ts-node ./scripts/reset-dependency-version node-fetch ^2.6.1`
2. call yarn in mapsdk
3. remove .npmrc in mapsdk
4. call yarn in mapsdk/coresdk
5. revert .npmrc in mapsdk
Finally you have two commits (in mapsdk and coresdk) that you can commit.

Reply:
ok, good to know.. thanks for the howto